scripts/check_delta.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/python3
+
```

Comment:
add header with license and copyright here, also a short text describing what this script does.